nibabel/tests/test_removalschedule.py

```diff
@@ -125,7 +125,7 @@ def test_module_removal():
     for module in _filter(MODULE_SCHEDULE):
         with pytest.raises(ImportError):
             __import__(module)
-            assert False, f'Time to remove {module}'
+            pytest.fail(f'Time to remove {module}')
```

Comment:
This breaks our test-testing test on L161.

Reply:
Changed into raise `AssertionError` instead.